data/Music/Sonic the Hedgehog Series/Sonic CD/You Can Do Anything.meta

```diff
@@ -0,0 +1,4 @@
+Sonic CD (JP) - You Can Do Anything
+0x21
+bgm
+TitleTheme
```

Comment:
The location line should have at least one group from each level of specificity (exact, high, mid and low).